### PR TITLE
added swagger

### PR DESCRIPTION
**Pull request by piotr-major-rmtcfm-com**

### CODE REVIEW

File: app/plugins/errorHandler.js [hunk 1]
```diff
@@ -37,6 +37,6 @@ module.exports = (err, request, response) => {
     console.error(err);
 
     return response
-        .status(StatusCodes.INTERNAL_SERVER_ERROR)
+        .sendStatus(StatusCodes.INTERNAL_SERVER_ERROR)
```
Comment (daniel-pupczyk-rmtcfm-com) [inline app/plugins/errorHandler.js]: it should be status

File: app/swagger-doc/requests/TodoBody.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+components:
+    requestBodies:
+        TodoBody:
+        description: A JSON object containing todo information
```
Comment (daniel-pupczyk-rmtcfm-com) [inline app/swagger-doc/requests/TodoBody.yml]: is it working? not needed to be nested?

File: app/swagger-doc/requests/TodoBody.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+        required: true
+        content:
+            application/json:
+                schema:
+                    $ref: '#/components/schemas/Todo'
```
Comment (daniel-pupczyk-rmtcfm-com) [inline app/swagger-doc/requests/TodoBody.yml]: are you sure? schemas/Todo should contain e.g. `createdAt` but request shouldnt contain `createdAt`

File: app/swagger-doc/requests/UserBody.yml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+components:
+    requestBodies:
+        UserBody:
+        description: A JSON object containing user information
```
Comment (daniel-pupczyk-rmtcfm-com) [inline app/swagger-doc/requests/UserBody.yml]: as above

File: app/swagger-doc/requests/UserBody.yml [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+        required: true
+        content:
+            application/json:
+                schema:
+                    properties:
+                        id:
```
Comment (daniel-pupczyk-rmtcfm-com) [inline app/swagger-doc/requests/UserBody.yml]: why do we need `id` in request body?

File: app/swagger-doc/schemas/User.yml [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+                    type: string
+                    description: User ID (primaryKey)
+                    format: uuid
+                userName:
+                    type: string
+                firstName:
+                    type: string
+                lastName:
+                    type: string
+                email:
```
Comment (daniel-pupczyk-rmtcfm-com) [inline app/swagger-doc/schemas/User.yml:16]: there is createdAt, updatedAt, etc

File: app/swagger-doc/schemas/Todo.yml [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+                    format: uuid
+                createdBy:
+                    type: string
+                    description: 'User.id (Foreign Key)'
+                    format: uuid
+                name:
+                    type: string
+                    example: 'test'
+                    description: 'User.id (Foreign Key)'
+                completed:
```
Comment (daniel-pupczyk-rmtcfm-com) [inline app/swagger-doc/schemas/Todo.yml:22]: as below

File: app/swagger-doc/routes/users.yml [hunk 8]
```diff
@@ -0,0 +1,108 @@
...
+                    $ref: '#/components/responses/BadRequestError'
+                '401':
+                    $ref: '#/components/responses/UnauthorizedError'
+                '403':
+                    $ref: '#/components/responses/ForbiddenError'
+        delete:
+            tags:
+                - user
+            summary: Delete user
+            description: ''
```
Comment (daniel-pupczyk-rmtcfm-com) [inline app/swagger-doc/routes/users.yml]: add description of each route

File: app/swagger-doc/routes/users.yml [hunk 9]
```diff
@@ -0,0 +1,108 @@
...
+        put:
+            tags:
+                - user
+            summary: Updated user
+            description: ''
+            operationId: updateUser
+            parameters:
+                - name: userId
+                  in: path
+                  description: id of user that need to be updated
```
Comment (daniel-pupczyk-rmtcfm-com) [inline app/swagger-doc/routes/users.yml]: start sentence with upper letter and end with `.`

File: app/swagger-doc/routes/users.yml [hunk 10]
```diff
@@ -0,0 +1,108 @@
...
+                    $ref: '#/components/responses/UnauthorizedError'
+                '403':
+                    $ref: '#/components/responses/ForbiddenError'
+    '/user/{userId}':
+        get:
+            tags:
+                - user
+            summary: Get user by user id
+            description: ''
+            operationId: getUserById
```
Comment (daniel-pupczyk-rmtcfm-com) [inline app/swagger-doc/routes/users.yml]: is it needed?

File: app/index.js [hunk 11]
```diff
@@ -3,7 +3,7 @@ const cors = require('cors');
 const bodyParser = require('body-parser');
 const bearerToken = require('express-bearer-token');
 const helmet = require('helmet');
-
+const path = require('path');
```
Comment (daniel-pupczyk-rmtcfm-com) [inline app/index.js]: one empty line after imports

File: app/index.js [hunk 12]
```diff
@@ -55,12 +56,15 @@ app.use(errorHandler);
...
 
+
+app.use('/swagger', express.static(path.join(__dirname, '../public/swagger')));
 app.use(
     '/',
     toggleAPIDocs,
     express.static(path.join(__dirname, '../public/api-doc'))
 );
 
+
```
Comment (daniel-pupczyk-prpl) [inline app/index.js:67]: why two lines?

File: app/index.js [hunk 13]
```diff
@@ -55,12 +56,15 @@ app.use(errorHandler);
 
 app.set('di', di);
 
+
```
Comment (daniel-pupczyk-prpl) [inline app/index.js:59]: why two lines?

File: app/swagger-doc/requests/TodoBody.yml [hunk 14]
```diff
@@ -0,0 +1,22 @@
...
+                        type: object
+                        properties:
+                            userId:
+                                type: string
+                                description: 'User.id (Foreign Key)'
+                                format: uuid
+                            name:
+                                type: string
+                                example: 'test'
+                                description: 'User.id (Foreign Key)'
```
Comment (daniel-pupczyk-prpl) [inline app/swagger-doc/requests/TodoBody.yml:18]: wrong description

**Pull request merged**